boilerplate/jest.config.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
   ...tsjPreset,
   preset: "jest-expo",
   transformIgnorePatterns: [
-    "<rootDir>/node_modules/(react-clone-referenced-element|@react-native-community|react-navigation|@react-navigation/.*|@unimodules/.*|native-base|react-native-code-push)",
+    "<rootDir>/node_modules/(?!((jest-)?react-native|@react-native(-community)?)|expo(nent)?|@expo(nent)?/.*|@expo-google-fonts/.*|react-navigation|@react-navigation/.*|@unimodules/.*|unimodules|sentry-expo|native-base|react-native-svg|react-clone-referenced-element|react-native-code-push)",
```

Comment:
to make it more readable here is a little hack I use

```
const packagesToTransform = [
  "react-native",
  "react-native-(.*)",
  "@react-native",
  "@react-native-community",
  "@react-navigation",
  "expo",
  "expo-(.*)",
  "react-native-svg",
  "@sentry/.*",
  "moti/.*",
  "@shopify/react-native-skia" // https://shopify.github.io/react-native-skia/docs/getting-started/installation#testing-with-jest
];
```

```
  transformIgnorePatterns: [
    `node_modules/(?!(${packagesToTransform.join("|")})/)`
  ],
```

Reply:
I like this a lot! So much more readable!